src/lang/defs/module.rs

```diff
@@ -10,7 +10,7 @@ use crate::lang::db::AnalysisDatabase;
 /// Information about the definition of a module.
 #[derive(Eq, PartialEq)]
 pub struct ModuleDef {
-    id: ModuleId,
+    pub id: ModuleId,
```

Comment:
to be consistent, avoid doing `pub` here and write a getter `pub fn module_id(&self) -> ModuleId`